lib/pavilion/schedulers/advanced.py

```diff
@@ -10,7 +10,7 @@
 from pavilion.status_file import STATES
 from pavilion.test_run import TestRun
 from pavilion.types import NodeInfo, Nodes, NodeList, NodeSet, NodeRange
-from .config import validate_config, AVAILABLE, BACKFILL
```

Comment:
calc_node_range was a static method (basically a function tacked on to a class, rather than a full method) attached to the scheduler plugin class. I moved it to the scheduler config module as a regular function. It's a better fit there anyway, and that let me use it to get the 'requested_nodes' as a scheduler variable consistent with how it was calculated elsewhere.